distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/licenses.xml

```diff
@@ -536,7 +536,7 @@
       <licenses>
         <license>
           <name>Apache Software License 2.0</name>
-          <url>https://raw.githubusercontent.com/webauthn4j/webauthn4j/0.10.2.RELEASE/LICENSE.txt</url>
+          <url>https://raw.githubusercontent.com/webauthn4j/webauthn4j/0.12.0.RELEASE/LICENSE.txt</url>
```

Comment:
The url is correct but can you please update the version tag to `0.12.0.RELEASE-redhat-00001`